### PR TITLE
LHDB-898 files re-organization

### PR DESCRIPTION
**Pull request by bernardyn**

### CODE REVIEW

**Comment by bernardyn** [comment]
Task linked: [LHDB-898 Reorganizacja plików w gicie w projekcie ontowikikul](https://app.clickup.com/t/2591891/LHDB-898)

File: Readme.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ See [Create entities](./create/README.md#create-entities)
 ## Templates:
 * [templates to create entities](https://github.com/ontohgis/wikikul/tree/main/create)
 
+## Serialization:
+To keep a consisteny of .ttl file syntax we are using a serialization script
+```
+lib/rdf-toolkit.sh ontowikikul.ttl
```
Comment (bernardyn) [inline Readme.md]: I am not sure if that command will works on linux/windows

**Review by przemekgradzki — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (przemekgradzki) [reply Readme.md]: Now works :+1:

File: ontowikikul.ttl
Comment (bernardyn) [inline ontowikikul.ttl:1]: @przemekgradzki I see you performed a serialization run as well. Of course it is fine, but I though we will update that file in separate task and PR.

**Pull request merged**